### DISCUSSION TITLE
netcdf compilation issue

### DISCUSSION THREAD

**Discussion by JMSnyder1980** [Unidata/netcdf-c#2640] [General]
I tried to compile netcdf-c-4.9.1 but got the following error:

../liblib/.libs/libnetcdf.so: undefined reference to H5allocate_memory' ../liblib/.libs/libnetcdf.so: undefined reference to H5free_memory'

How do I resolve this error?

**Comment by WardF** [comment]
Can you provide the `config.log` generated when you configured your build?

**Comment by JMSnyder1980** [comment]
Attached are my configuration files for my build including the config.log:


[config.log](https://github.com/Unidata/netcdf-c/files/10847215/config.log)
[config.status.txt](https://github.com/Unidata/netcdf-c/files/10847220/config.status.txt)
[config.sub.txt](https://github.com/Unidata/netcdf-c/files/10847222/config.sub.txt)
[configure.ac.txt](https://github.com/Unidata/netcdf-c/files/10847224/configure.ac.txt)
[configure.txt](https://github.com/Unidata/netcdf-c/files/10847226/configure.txt)

**Reply by JMSnyder1980** [reply]
If I compile this along with the fortran package will I be able to generate the netdf.mod file?

**Comment by WardF** [comment]
Thanks for providing the `config.log` file.  It shows an error when running `configure`, e.g. `exit 1`, which makes me wonder how you were able to then run `make`, since the `Makefiles` weren't properly generated.  The original issue you describe above suggests an issue with the runtime linker, which would happen post-configuration.  Are you certain this is the correct `config.log` file?
